core/config/capabilities_config.go

```diff
@@ -15,6 +16,7 @@ type CapabilitiesWorkflowRegistry interface {
 	Address() string
 	NetworkID() string
 	ChainID() string
+	MaxArtifactsSize() utils.FileSize
```

Comment:
You need multiple values here
MaxSecretsFileSize
MaxConfigFileSize
MaxBinaryFileSize